read-fonts/src/tables/truetype/scale.rs

```diff
@@ -0,0 +1,718 @@
+//! Scaler for TrueType outlines.
+
+use super::{
+    deltas, Error, HintOutline, Outline, OutlineInfo, OutlineMemory, COMPOSITE_RECURSION_LIMIT,
+    PHANTOM_POINT_COUNT,
+};
+use crate::{
+    tables::{
+        glyf::{
+            Anchor, CompositeGlyph, CompositeGlyphFlags, Glyf, Glyph, PointMarker, SimpleGlyph,
+        },
+        gvar::Gvar,
+        hmtx::Hmtx,
+        hvar::Hvar,
+        loca::Loca,
+    },
+    types::{F26Dot6, F2Dot14, Fixed, GlyphId, Point},
+    TableProvider,
+};
+
+/// Scaler state for TrueType outlines.
```

Comment:
Assuming this is all actually public API I think it would benefit from a bit more documentation around how exactly the consumer should use it.